test/multiverse/lib/multiverse/envfile.rb

```diff
@@ -112,6 +112,11 @@ def add_twiddle_wakka(version)
       ", '~> #{version}'"
     end
 
+    def add_version(version)
```

Comment:
This is so close to `add_twiddle_waka` that I wonder if we ought to just combine them:

```ruby
def add_version(version, twiddle_wakka: true)
  return unless version  # alternative way to write this guard clause

  ", '#{'~> ' if twiddle_wakka}#{version}'"
end
```

then all existing `add_twiddle_waka` calls could be renamed to `add_version` without any other changes.